### PR TITLE
initial setup of styleguidist

### PR DESCRIPTION
**Pull request by kevinbarabash**
This PR does the initial set up of styleguidist.  It doesn't handle snapshot tests yet.  This will be handled by a separate PR using https://github.com/styleguidist/snapguidist.  This PR doesn't include Button yet in the style guide because it's not ready yet.

screen shot 2018-04-17 at 2 48 31 pm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "test": "npm run flow && npm run jest",
     "coverage": "STORYBOOK_INLINE_APHRODITE=1 jest --coverage",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "build:storybook": "build-storybook -c .storybook -o docs && git add docs"
+    "styleguidist": "styleguidist server",
+    "build:styleguidist": "styleguidist build"
```
Comment (jeresig) [inline package.json]: Could we add back in the `&& git add docs`? Alternatively you can add a new command like `"git-add-docs": "git add docs",` and then add `"git-add-docs"` to the `"pre-commit"` array below.

File: packages/wonder-blocks-typography/index.js [hunk 2]
```diff
@@ -1,5 +1,40 @@
 // @flow
```
Comment (jeresig) [inline packages/wonder-blocks-typography/index.js]: So we just ignore index.js files, right?

File: styleguide.config.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+const {createConfig, babel, postcss} = require("webpack-blocks");
+
+module.exports = {
+    webpackConfig: createConfig([babel(), postcss()]),
```
Comment (jeresig) [inline styleguide.config.js]: Interesting! Hadn't seen this before. Are we using postcss? I'm not sure I see it anywhere?

Comment (kevinbarabash) [reply styleguide.config.js]: I just copy pasted from the styleguidist quick start.

File: package.json (hunk 1 above)
Comment (kevinbarabash) [reply package.json]: Ah... I missed that.  Will do.

File: packages/wonder-blocks-typography/index.js (hunk 2 above)
Comment (kevinbarabash) [reply packages/wonder-blocks-typography/index.js]: `styleguidist` ignores the `index.js` files.  They're still the entry points though for the modules.

**Pull request merged**